File: Data Structures & Algorithms/valid-palindrome-ii/submission-0.java
class Solution {
    public boolean validPalindrome(String s) {
        char[] arr = s.toCharArray();
        int l = 0, r = arr.length - 1;
        int count = 1;
        while(l < r){
            if(arr[l] == (arr[r])){
                l++;
                r--;
            } else{
                if(arr[l + 1] == (arr[r]) || arr[l] == (arr[r - 1])){ 
                    l++;
                    r--;
                    count--; 
                }else return false;
                if(count < 0) return false;
            }
        }
        return true;
    }
}

File: Data Structures & Algorithms/length-of-last-word/submission-0.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        int rs = 0;
        for(int i  = s.Length - 1; i >= 0; i--){
            if(s[i] == ' ' && rs != 0) return rs;
            if(s[i] != ' ') rs++;
        }
        return rs;
    }
}